__testcafe__/peruse.spec.ts

```diff
@@ -76,6 +77,20 @@ test( 'can close a tab', async ( t ) => {
         .eql( 1 );
 } );
 
+test( 'can reopen a tab', async ( t ) => {
+    await t
+        .click( `.${CLASSES.ADD_TAB}` )
+        .expect( Selector( `.${CLASSES.TAB}` ).count )
+        .eql( 2 )
+        .click( `.${CLASSES.CLOSE_TAB}` )
+        .expect( Selector( `.${CLASSES.TAB}` ).count )
+        .eql( 1 );
+
```

Comment:
Some checks on the tab/address bar contents are worthwhile here, I think. All we're checking now is that there are 2 tabs. Not that the one we just closed was reopened...